app/components/SubscriptionTable/SubscriptionTable.tsx

```diff
@@ -35,7 +35,10 @@ const SubscriptionsTable: FC<Props> = (props) => {
       <table>
         <thead>
           <tr>
-            <th>OrderId</th>
+            <th>OrderId
+

```

Comment:
@Mohamed-Khider you need to always go through the changes in your PR and ensure that they are supposed to be there, for example. This change here clearly adds no value. Also the styling changes are not helpful- you cannot keep changing the whole project to you local style when you join a project- please change your local set-up accordingly to fix this.